doc/install/advanced.rst

```diff
@@ -113,10 +98,13 @@ like Anaconda, Miniconda includes the ``conda`` command line tool for
 installing new packages and managing environments; unlike Anaconda, Miniconda
 starts off with a minimal set of around 30 packages instead of Anaconda's
 hundreds. See the `installation instructions for Miniconda`_ for more info.
+A similar alternative is `MiniForge`_, which uses the ``conda-forge`` channel
+as the default source for package installation (saving you the trouble of
+typing ``--channel conda-forge`` with each ``conda install`` command).
 
-It is also possible to use a system-level installation of Python (version 3.6
-or higher) and use ``pip`` to install MNE-Python and its dependencies, using
-the provided `requirements file`_:
+It is also possible to use a system-level installation of Python (version
+|min_python_version| or higher) and use ``pip`` to install MNE-Python and its
+dependencies, using the provided `requirements file`_:
```

Comment:
Yes! IMO every proper Python package needs to be `pip` installable. I wouldn't completely switch to `conda`.

Reply:
@hoechenberger this is in the advanced docs under "other python distros" so it's not an encouragement. I think it's entirely appropriate to say "you can do a system-install and it will probably work". By this point in the install docs it should be clear that it's not what we recommend.